static/js/biomedical/disease/page.tsx

```diff
@@ -65,6 +68,8 @@ export class Page extends React.Component<PagePropType, PageStateType> {
     );
   }
   render(): JSX.Element {
+    const diseaseName = getDiseaseCommonName(this.state.data);
```

Comment:
Consider a fallback to the DCID if there is no name available (or there was an error parsing / fixing the name)

Reply:
That's a great idea! Added a fallback to the DCID!